bazel-java-deps.bzl

```diff
@@ -44,8 +30,9 @@ netty_version = "4.1.67.Final"
 # <tcnative.version>2.0.42.Final</tcnative.version>
 # ```
 netty_tcnative_version = "2.0.40.Final"
-grpc_version = "1.41.0"
+grpc_version = "1.35.0"
```

Comment:
@hubert-da you didn’t actually repin after changing that. The lock file still pointed to this. I changed it back for now since the upgrade breaks some stuff. Feel free to actually upgrade it again, I just don’t want to deal with the breakage in this PR.

Reply:
Just got to it, sorry, will do it again.